hack/generate-manifest.sh

```diff
@@ -84,6 +86,9 @@ MULTICAST=false
 MULTICAST_INTERFACES=""
 HELM_VALUES_FILES=()
 HELM_VALUES=()
+ENABLE_NODEPORT=false
+NODEPORT_NUMBER=32767
```

Comment:
names are too generic

Reply:
Updated it. is this better?
EXPORT_CONTROLLER_SERVICE=false
NODEPORT=32767

Reply:
I think you only need one variable (`ANTREA_SERVICE_NODEPORT`) with 0 meaning "NodePort disabled"

Reply:
I have updated it.